packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

```diff
@@ -179,10 +184,10 @@ contract L2StandardBridge is StandardBridge, Semver {
         uint32 _minGasLimit,
         bytes memory _extraData
     ) internal {
-        address l1Token = OptimismMintableERC20(_l2Token).l1Token();
```

Comment:
I don't like the idea of moving this. It confirms that when withdrawing ETH you are referencing the correct token pair.

Reply:
I don't understand what you mean here because the branch ensures that as well?

Reply:
This specifically guarantees that when using this codepath to withdraw, you specify (OVM_ETH, FakeL1EthTokenAddress) as the token pair and you cannot use (OVM_ETH, RandomToken).